framework/core/js/src/admin/components/HeaderPrimary.tsx

```diff
@@ -20,10 +22,9 @@ export default class HeaderPrimary extends Component {
 
   /**
    * Build an item list for the controls.
-   *
-   * @return {ItemList<import('mithril').Children>}
    */
   items() {
-    return new ItemList();
+    const items = new ItemList<Mithril.Children>();
+    return items;
```

Comment:
let's add an explicit return type to this method and the other items method changed in this PR